target/linux/ramips/image/mt7621.mk

```diff
@@ -335,6 +335,18 @@ define Device/iodata_wnpr2600g
 endef
 TARGET_DEVICES += iodata_wnpr2600g
 
+define Device/iptime_a8004t
+  MTK_SOC := mt7621
+  IMAGE_SIZE := 16064k
+  UIMAGE_NAME := a8004t
+  DEVICE_VENDOR := ipTIME
+  DEVICE_MODEL := A8004T
+  DEVICE_PACKAGES := \
+	kmod-mt7615e kmod-usb3 kmod-usb-ledtrig-usbport wpad-basic
+  SUPPORTED_DEVICES += iptime_a8004t
```

Comment:
Remove that line. It's only required if you update from an older version of OpenWrt with that particular board name.